README.md

```diff
@@ -128,18 +128,19 @@ Alaska team
 * [Alaska organizational chart](https://app.mural.co/t/dhssalaska0106/m/dhssalaska0106/1576183633753/83e37044b807aac3ae09c6757d794e45dd931ea6) (includes product team members)
 * [Alaska Department of Health and Social Services organizational chart](http://dhss.alaska.gov/Commissioner/Documents/PDF/DHSS_org_chart.pdf) (official departmental org chart)
 
-EIS-R project management
+EIS-M project management
 * This repo - This repository serves as the overarching umbrella repo for the EIS-R project
```

Comment:
- Update EIS-R to EIS-M
- Think that maybe we should include after the sentence... . Please note that each solicitation will have its own GitHub repository.